tensorflow_datasets/audio/commonvoice.py

```diff
@@ -0,0 +1,126 @@
+    "ky": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/ky.tar.gz",
+    "ga-IE": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/ga-IE.tar.gz",
+    "kab": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/kab.tar.gz",
+    "ca": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/ca.tar.gz",
+    "zh-TW": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/zh-TW.tar.gz",
+    "sl": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/sl.tar.gz",
+    "it": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/it.tar.gz",
+    "nl": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/nl.tar.gz",
+    "cnh": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/cnh.tar.gz",
+    "eo": "https://voice-prod-bundler-ee1969a6ce8178826482b88e843c335139bd3fb4.s3.amazonaws.com/cv-corpus-1/eo.tar.gz"}
+_SPLITS = {
+    tfds.Split.TRAIN: "train",
+    tfds.Split.TEST: "test",
+    tfds.Split.VALIDATION: "validated"}
+
+
+class CommonVoiceConfig(tfds.core.BuilderConfig):
+    """
+        Configuration Class for Mozilla CommonVoice Dataset
+    """
+    @tfds.core.api_utils.disallow_positional_args
+    def __init__(self, language="en", **kwargs):
+        if language not in _DOWNLOAD_LINKS.keys():
+            raise ValueError(
+                "language must be one of %s" %
+                _DOWNLOAD_LINKS.keys())
+        self._language = language
+        name = kwargs.get("name", None)
+        name = "%s_%s" % ("common_voice" if name is None else name, language)
+        kwargs["name"] = name
+        description = kwargs.get("description", None)
+        description = "Language Code: %s" % language if description is None else description
+        kwargs["description"] = description
+        super(CommonVoiceConfig, self).__init__(**kwargs)
+
+    @property
+    def download_urls(self):
+        """
+            Property returning Download URL based on Language Specified
+        """
+        return _DOWNLOAD_LINKS[self._language]
+
+
+def _generate_builder_configs():
+    """
+        Generates Builder Configs
+    """
+    configs = []
+    version = "1.0.0"
+    for k in _DOWNLOAD_LINKS.keys():
+        config = CommonVoiceConfig(version=version, language=k)
+        configs.append(config)
+    return configs
+
+
+class CommonVoice(tfds.core.GeneratorBasedBuilder):
+    """
+    Mozilla Common Voice Dataset
+    """
+    BUILDER_CONFIGS = _generate_builder_configs()
+    VERSION = tfds.core.Version("1.0.0")
+
+    def _info(self):
+        return tfds.core.DatasetInfo(
```

Comment:
Could you please update the formatting to use 2 spaces instead of 4 for indentation?

Reply:
Resolved